### PR TITLE
feat(Animeflix): add presence

### PR DESCRIPTION
**Pull request by Micotsu**
Pull request to add my site https://animeflix.live.

### CODE REVIEW

**Comment by shiseni** [comment]
You need `yarn format`

File: websites/A/Animeflix/presence.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+	});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+    largeImageKey: "https://i.imgur.com/4hiyxuW.png"
+  }, { pathname } = document.location;
+
+  console.log(pathname)
```
Comment (shiseni) [inline websites/A/Animeflix/presence.ts:14]: Remove console.log.

File: websites/A/Animeflix/presence.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  if (pathname === "/") presenceData.details = "Exploring Main Page";
+  else if(pathname === "/genres") presenceData.details = "Exploring Genres";
```
Comment (shiseni) [inline websites/A/Animeflix/presence.ts:16]: You can add a state. For example, when the anime information window appears, display the title. This also applies to the following lines.

File: websites/A/Animeflix/presence.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+const presence = new Presence({
+		clientId: "1039178085922250873",
+	}),
+	strings = presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+	});
```
Comment (shiseni) [inline websites/A/Animeflix/presence.ts:7]: You need to get strings like in this example #6973

File: websites/A/Animeflix/presence.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  else if(pathname === "/genres") presenceData.details = "Exploring Genres";
+  else if(pathname === "/series") presenceData.details = "Exploring Series";
+  else if(pathname === "/movies") presenceData.details = "Exploring Movies";
+  else if(pathname === "/mylist") presenceData.details = "Going through My List";
+  else if(pathname.includes("/watch")) {
+    const showName = <HTMLMetaElement>document.querySelector('meta[name="anime-skip.show.name"]');
+    const episodeName = <HTMLMetaElement>document.querySelector('meta[name="anime-skip.episode.name"]');
+    const epNum = <HTMLMetaElement>document.querySelector('meta[name="anime-skip.episode.number"]');
+    presenceData.details = `Watching ${showName.content}`
+    presenceData.state = episodeName.content.includes("Episode") ? episodeName.content : `${epNum.content}. ${episodeName.content}`
```
Comment (shiseni) [inline websites/A/Animeflix/presence.ts:25]: Here you can add video playback duration and a small image with action (play/pause).

File: websites/A/Animeflix/presence.ts (hunk 1 above)
Comment (shiseni) [inline websites/A/Animeflix/presence.ts:14]:
```suggestion
```

File: websites/A/Animeflix/presence.ts (hunk 3 above)
Comment (shiseni) [inline websites/A/Animeflix/presence.ts:7]: You need to get strings like in this example https://github.com/PreMiD/Presences/blob/5e5f99668ba4800524e66c2a3fc9fd38755025c6/websites/S/Skribbl.io/presence.ts#L14

File: websites/A/Animeflix/presence.ts (hunk 2 above)
Comment (shiseni) [inline websites/A/Animeflix/presence.ts:16]: You can add a state. For example, when the anime information window appears, display the title. This also applies to the following lines.

File: websites/A/Animeflix/presence.ts (hunk 4 above)
Comment (shiseni) [inline websites/A/Animeflix/presence.ts:25]: Here you can add video playback duration and a small image with action (play/pause).

File: websites/A/Animeflix/presence.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  console.log(pathname)
+  if (pathname === "/") presenceData.details = "Exploring Main Page";
+  else if(pathname === "/genres") presenceData.details = "Exploring Genres";
+  else if(pathname === "/series") presenceData.details = "Exploring Series";
+  else if(pathname === "/movies") presenceData.details = "Exploring Movies";
+  else if(pathname === "/mylist") presenceData.details = "Going through My List";
+  else if(pathname.includes("/watch")) {
+    const showName = <HTMLMetaElement>document.querySelector('meta[name="anime-skip.show.name"]');
+    const episodeName = <HTMLMetaElement>document.querySelector('meta[name="anime-skip.episode.name"]');
+    const epNum = <HTMLMetaElement>document.querySelector('meta[name="anime-skip.episode.number"]');
```
Comment (shiseni) [inline websites/A/Animeflix/presence.ts:23]:
```suggestion
    const showName = document.querySelector<HTMLMetaElement>('meta[name="anime-skip.show.name"]');
    const episodeName = document.querySelector<HTMLMetaElement>('meta[name="anime-skip.episode.name"]');
    const epNum = document.querySelector<HTMLMetaElement>('meta[name="anime-skip.episode.number"]');
```

**Comment by Micotsu** [comment]
Thanks for the help! I didn't feel the need for any extra features as my users and me are happy with the current format. I might add some more later.

File: websites/A/Animeflix/metadata.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "326694667003363328",
+		"name": "Micotsu#2907"
```
Comment (theusaf) [inline websites/A/Animeflix/metadata.json:5]: Discriminator number is a mistake caused by an old version of `pmd`
```suggestion
		"name": "Micotsu"
```

File: websites/A/Animeflix/metadata.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+	},
+	"service": "Animeflix",
+	"description": {
+		"en": "Unlimited anime without ads!"
+	},
+	"url": "animeflix.live",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/4hiyxuW.png",
```
Comment (theusaf) [inline websites/A/Animeflix/metadata.json:13]: Logos should be at least 512x512px.

File: websites/A/Animeflix/metadata.json [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+	"thumbnail": "https://i.imgur.com/XmDNhW4.png",
```
Comment (theusaf) [inline websites/A/Animeflix/metadata.json:14]: This thumbnail may be a bit small/low-quality, but I'm not sure if it *needs* to be replaced.

File: websites/A/Animeflix/metadata.json [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "326694667003363328",
+		"name": "Micotsu#2907"
+	},
+	"service": "Animeflix",
+	"description": {
+		"en": "Unlimited anime without ads!"
+	},
+	"url": "animeflix.live",
```
Comment (theusaf) [inline websites/A/Animeflix/metadata.json:11]: Site supports `www` 'subdomain'.
```suggestion
	"url": ["animeflix.live", "www.animeflix.live"],
```

File: websites/A/Animeflix/presence.ts [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+			largeImageKey: "https://i.imgur.com/4hiyxuW.png",
+			buttons: [
+				{
+					label: "Visit Website",
+					url: "https://animeflix.live/",
+				},
+				{
+					label: "Join Discord",
+					url: "https://discord.gg/SKcb2C7HjH",
+				},
```
Comment (theusaf) [inline websites/A/Animeflix/presence.ts:16]: Buttons that lead to home pages are not allowed.

I'm also not sure if buttons to Discord servers are allowed.

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by EncryptedDev** [comment]
Title doesn't follow commit convention

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**